projects/extension/sql/idempotent/013-loading.sql

```diff
@@ -0,0 +1,83 @@
+-------------------------------------------------------------------------------
+-- loading_row
+create or replace function ai.loading_row
+( column_name pg_catalog.text)
```

Comment:
```suggestion
( content_column pg_catalog.name)
```
maybe? It better describes which column we expect to be provided.

Also, use the `name` type

Reply:
I didn't rename the parameter name I feel like with `loading_column(column_name => '')` makes a lot of sense.